### ISSUE TITLE
ability to see last months map so you can see who you talked to last time i.e. unique event urls

### ISSUE THREAD

**Issue by djay** [bkkhack/hackmap#7]
maybe http://hackmap.com/bkkhack/events/7
and http://hackmap.com/bkkhack/events redirects based on the label?

Implicit in the above urls is if the repo name should be in the url. Pro: allows the code to reused for any repo without forking. con: allows the code to reused with any repo without forking, could have problems with oauth? :)

**Comment by djay** [comment]
@waf if the same code is used for many events/repos, what are the implications of oauth?

**Comment by waf** [comment]
@djay The oauth application registered with GitHub needs a single [callback url that must match exactly](https://developer.github.com/v3/guides/basics-of-authentication/#registering-your-app). So each event/repo would need its own application, and its own endpoint running [gatekeeper](https://github.com/prose/gatekeeper)

**Comment by djay** [comment]
@waf can't people login to http://hackmap.com/bkkhack and then we redirect after that?
Otherwise it would be really restrictive. 
Or else we could do http://hackmap.com/bkkhack?map=20170801&userid=djay or http://hackmap.com/bkkhack#!20170801/djay

**Comment by djay** [comment]
@waf This discussion kind of got side tracked... but the need is still there.
What would be super useful is to send out a url to everyone saying here is last month, see what interesting things people did and here is another url for this month...

**Comment by waf** [comment]
@djay I just merged https://github.com/bkkhack/hackmap/pull/62 which fixed the current issues with the routing, and deployed it.

Example URLs:

- https://bkkhack.github.io/hackmap/ - the current hacknight (currently March)
- https://bkkhack.github.io/hackmap/#/events/43 - the hacknight with github issue 43 (February)

See the PR for the explanation about the hash in the URL. 

Earlier hacknights, which relied on us editing the source code to update the floorplan image, don't work. The later hacknights, where we embedded the map url in the issue comment, do work.
